docs/source/usage/Ellipsoid.rst

```diff
@@ -0,0 +1,117 @@
+Ellipsoid  transformation
+================================================================================
+A ellipsoid is the mathematically defined surface which approximates the geiod, surface of the earth's gravity field, which is approximately the same as mean sea level.
+
+
+.. figure:: ./images/General_ellpsoid.png
+   :width: 500 px
+   :align: center
+   :alt:   Global and local fitting of the ellipsoid
+
+   proj-string: ``+proj=merc``
+
+
+
+A complete ellipsoid definition comprises a size (primary) and a shape (secondary)
+parameter.
+
+Ellipsoid Size parameters 
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+.. _ellipsoid_parameters : PROJ/docs/source/operations/options/
+
+	============   ==============================================================
+	Parameter      Description
+	============   ==============================================================
+	+R             Radius of a spherical planet
+	+a             Semi-major axis of an ellipsoidal planet
+	============   ==============================================================
+
+Ellipsoid Shape parameters 
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+.. _ellipsoid_parameters : PROJ/docs/source/operations/options/
+
+	============   ==============================================================
+	Parameter      Description
+	============   ==============================================================
+	+rf            Reverse flattening of the ellipsoid
+	+f             Flattening of the ellipsoid
+	+es            Eccentricity squared
+	+e             Eccentricity
+	+b             Semi-minor axis
+	============   ==============================================================
+
+The ellipsoid definition may be augmented with a spherification flag, turning
+the ellipsoid into a sphere with features defined by the ellipsoid.
+
+Ellipsoid Spherification parameters 
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+.. _ellipsoid_parameters : PROJ/docs/source/operations/options/
+
+	============   ==============================================================
+	Parameter      Description
+	============   ==============================================================
+	+R_A           A sphere with the same surface area and volume as the ellipsoid
+	+R_a           A sphere with R = (a + b)/2   (arithmetic mean)
+	+R_g           A sphere with R = sqrt(a*b)   (geometric mean)
+	+R_h           A sphere with R = 2*a*b/(a+b) (harmonic mean)
+	+R_lat_a=phi   A sphere with R being the arithmetic mean of the corresponding ellipsoid at latitude phi.
+	+R_lat_g=phi   A sphere with R being the geometric mean of the corresponding ellipsoid at latitude phi.
+	============   ==============================================================
+
+If R is given as size parameter, any shape and spherification parameters
```

Comment:
Now that we are using option directives we can format and link to options in a nice way:

```suggestion
If :option:`+R` is given as size parameter, any shape and spherification parameters
```

There are several other places in the text where this should be done.